cmd/root.go

```diff
@@ -59,4 +61,5 @@ func init() {
 	rootCmd.Flags().StringVar(&messageBroker, "message-broker", "", fmt.Sprintf("Publish resource state changes to a message broker (supported values: %v)", message.SupportedBrokers))
 	rootCmd.Flags().StringVar(&amqpURI, "amqp-uri", "", "AMQP broker connection URI")
 	rootCmd.Flags().StringVar(&amqpQueue, "amqp-queue", "", "AMQP broker queue name")
+	rootCmd.Flags().StringVar(&preparerPluginsPath, "preparer-plugins-path", "", "Path to specific preparer plugins or directory to load them from")
```

Comment:
@steved 

I wonder if we want to use something like `XDG_DATA_HOME/plugins` as a default here? We could then just install plugins into the default path and have them picked up. Thoughts?

Reply:
Yep, that makes sense